src/plone/restapi/services/content/configure.zcml

```diff
@@ -70,6 +70,45 @@
     permission="zope2.View"
     />
 
+  <plone:service
+    method="OPTIONS"
+    name="@upload"
```

Comment:
@buchi here are my notes from our discussion at Beethoven sprint:

https://github.com/plone/plone.restapi/issues/79#issuecomment-287565245

I agree that it doesn't make sense to have two upload endpoints if TUS is all we have. Though we have to make sure we keep our API in sync with Guillotina.